internal/component/prometheus/exporter/postgres/postgres.go

```diff
@@ -30,37 +28,6 @@ func createExporter(opts component.Options, args component.Arguments, defaultIns
 	return integrations.NewIntegrationWithInstanceKey(opts.Logger, a.convert(opts.ID), defaultInstanceKey)
 }
 
-func parsePostgresURL(url string) (map[string]string, error) {
```

Comment:
This was not used and a duplicate of https://github.com/grafana/alloy/blob/main/internal/static/integrations/postgres_exporter/postgres_exporter.go#L90